example_byteslice_test.go

```diff
@@ -8,3 +8,10 @@ func ExampleReverse() {
 	fmt.Printf("%x\n", Reverse(data))
 	// Output: bada55
 }
+
+func Example() {
```

Comment:
A documentation example function needs to be `Example<functionName>` in your case `ExampleLPad`